typhos/display.py

```diff
@@ -1287,6 +1288,16 @@ def size_hint(*args, **kwargs):
 
         self._update_children()
         utils.reload_widget_stylesheet(self)
+        self.updateGeometry()
+        self.template_changed.emit(template)
+
+    def minimumSizeHint(self) -> QtCore.QSize:
+        if self._scroll_area is None:
+            return super().minimumSizeHint()
+        return QtCore.QSize(
+            self._scroll_area.viewportSizeHint().width(),
+            self.sizeHint().height(),
```

Comment:
Does this implicitly map `minimumSizeHint`'s height to `sizeHint` in these cases? What's the effect of that (over using e.g. `super().minimumSizeHint().height()`

Reply:
I had found that the `minimumSizeHint` was really small in my testing for the display (like 100px or so), switched it to `sizeHint` and forgot to loop back.

`minimumSizeHint` is probably correct here. I'll switch it back and see if things are still good or if we need to re-evaluate

Reply:
After switching it back, it still looks good to me. I think it didn't affect much because:
1. The dimension in question the height (where we're only resizing the width)
2. The size hint certainly is hit when laying out the device display in a scroll box, but I'm not sure about the min size hint

Reply:
Gotcha, that makes sense to me.